.editorconfig

```diff
@@ -202,3 +202,7 @@ end_of_line = crlf
 [src/**/*.{cs,vb}]
 # IDE0005: Remove unnecessary usings/imports
 dotnet_diagnostic.IDE0005.severity = warning
+
+[.nuget/**/*.cs]
+# Instantiate argument exceptions correctly
+dotnet_diagnostic.CA2208.severity = suggestion
```

Comment:
Should this be `dotnet_diagnostic.severity = suggestion`?

Reply:
I'm disabling CA2208 specifically so that you can clearly see what rules are violated by the referenced packages.